deps/rabbit/src/rabbit_channel_tracking.erl

```diff
@@ -236,19 +172,10 @@ count_tracked_items_in_ets({user, Username}) ->
 count_local_tracked_items_of_user(Username) ->
     rabbit_tracking:read_ets_counter(?TRACKED_CHANNEL_TABLE_PER_USER, Username).
 
-count_tracked_items_in_mnesia({user, Username}) ->
-    rabbit_tracking:count_tracked_items_mnesia(
-        fun tracked_channel_per_user_table_name_for/1,
-        #tracked_channel_per_user.channel_count, Username,
-        "channels of user").
-
 -spec clear_tracking_tables() -> ok.
 
 clear_tracking_tables() ->
-    case rabbit_feature_flags:is_enabled(tracking_records_in_ets) of
-        true -> ok;
-        false -> clear_tracked_channel_tables_for_this_node()
-    end.
```

Comment:
Can we now delete function `clear_tracking_tables/0` and also delete its callback declaration in `rabbit_tracking` ?